### PR TITLE
Toolchain refactor release

### PR DESCRIPTION
**Pull request by lrvick**

### CODE REVIEW

File: config/make.env [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+AWS_NITRO_DRIVER_HASH=4a9dab5aff4fa6a9f6b1e8e7cf284791726e0dde99e608f16768baa72aa11ba7
+
+PCSC_URL=https://codeload.github.com/LudovicRousseau/PCSC/legacy.tar.gz/15c16c7796607b1c8a2ce253d3f536918ab26b4a
+PCSC_HASH=24e8dfca5bfc6b282cfad6908d71bef6ba0e126df7efdf1200bc4ae41bd726e0
+
+OPENSSL_URL=https://codeload.github.com/openssl/openssl/legacy.tar.gz/a92271e03a8d0dee507b6f1e7f49512568b2c7ad
+OPENSSL_HASH=3ed44ac73cd2ff7a9928eead129696734fe02692a277210c3afa18b4999eb2cf
+
+#RUST_URL=https://static.rust-lang.org/dist/rustc-1.63.0-src.tar.gz
+#RUST_HASH=1f9580295642ef5da7e475a8da2397d65153d3f2cb92849dbd08ed0effca99d0
```
Comment (emostov) [inline config/make.env:19]: do we want to delete these?

File: Makefile [hunk 2]
```diff
@@ -1,409 +1,38 @@
...
-	$(OUT_DIR)/qos_client.$(ARCH).tar \
+	$(OUT_DIR)/qos_client.x86_64.tar \
 	$(OUT_DIR)/qos_host.linux-x86_64 \
 	$(OUT_DIR)/qos_host.oci.x86_64.tar \
-	$(OUT_DIR)/qos_host.$(ARCH).tar \
+	$(OUT_DIR)/qos_host.x86_64.tar \
 	$(OUT_DIR)/qos_enclave.linux-x86_64 \
 	$(OUT_DIR)/qos_enclave.oci.x86_64.tar \
-	$(OUT_DIR)/qos_enclave.$(ARCH).tar \
+	$(OUT_DIR)/qos_enclave.x86_64.tar \
```
Comment (emostov) [inline Makefile:25]: any specific reason you took out arch from here?

File: src/make/fetch.mk [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+$(FETCH_DIR)/pcsc.tar:
+	$(call fetch_file,PCSC_URL,PCSC_HASH)
+
+$(FETCH_DIR)/openssl.tar:
+	$(call fetch_file,OPENSSL_URL,OPENSSL_HASH)
+
+#//// OLD
+
+
```
Comment (emostov) [inline src/make/fetch.mk]: extra newline

File: src/make/rust.mk [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+		&& cd $(SRC_DIR)/qos_host \
+		&& cargo build \
+			--features vm \
+			$(CARGO_FLAGS) \
+		&& cp \
+			../target/x86_64-unknown-linux-musl/release/qos_host \
+			/home/build/$@; \
+	")
+
+
```
Comment (emostov) [inline src/make/rust.mk]: extra newline

File: Makefile (hunk 2 above)
Comment (lrvick) [reply Makefile:25]: if you were to pass an ARCH arg other than x86_64 we do not support it atm, so instead including only a full set of architectures we want to always build for here, which today is only one. see the pattern in tkcli makefile.

File: config/make.env (hunk 1 above)
Comment (lrvick) [reply config/make.env:19]: We pull this from git now below

**Pull request closed without merging**